handshake_server.go

```diff
@@ -356,6 +369,24 @@ Curves:
 		}
 	}
 
+	if hs.clientHello.delegatedCredentials {
```

Comment:
Ah, alright, I simply set the function in the config when spinning up TLS, right?